config.md

```diff
@@ -34,6 +34,33 @@ Each container has exactly one *root filesystem*, specified in the *root* object
 }
 ```
 
+## Mount Points
+
+You can add array of mount points inside container as `mounts`.
+Each record in this array must have configuration in [runtime config](runtime-config.md#mount-configuration).
+
+* **name** (string, required) Name of mount point. Used for config lookup.
+* **path** (string, required) Destination of mount point: path inside container.
+
+*Example*
+
+```json
+"mounts": {
+    "name": "proc",
+    "path": "/proc",
+}
+```
+In `runtime.json`:
```

Comment:
Ah, I wasn't suggesting we copy the `runtime.json` side over here.  I want to [*link* to it][1].  But I think the `config.json` example here should be compatible with the `runtime.json` example we link to.  I'm happy to PR a fixup against your branch if that's easier ;).

[1]: https://github.com/opencontainers/specs/pull/136/files#diff-695bac0ce528e5c6765a6666fcccc425R14